File: app/src/main/java/com/example/billeteramodulo4/Login_page.java
package com.example.billeteramodulo4;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageButton;

import androidx.appcompat.app.AppCompatActivity;

public class Login_page extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.login_page);


        // Abrir nueva activity

        ImageButton boton2 = findViewById(R.id.sendhome);

        boton2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

                //ir a activity 3
                Intent i = new Intent(Login_page.this, Signup_Page.class);
                startActivity(i);
            }
        });


    }
}

File: app/src/main/java/com/example/billeteramodulo4/LoginPage.kt
package com.example.billeteramodulo4

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import androidx.fragment.app.viewModels
import androidx.navigation.fragment.findNavController
import com.example.billeteramodulo4.databinding.FragmentLoginPageBinding
import com.example.billeteramodulo4.viewmodel.LoginPageViewModel

class LoginPage : Fragment() {

    private lateinit var binding: FragmentLoginPageBinding

    private val viewModel: LoginPageViewModel by viewModels()

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View {

        binding = FragmentLoginPageBinding.inflate(inflater, container, false)
        val root = binding.root

        binding.crearUnaNuevaCuentaLogin.setOnClickListener {

            findNavController().navigate(R.id.action_loginPageFragment_to_singupPageFragment)

        }

        binding.btnLogin.setOnClickListener {
            findNavController().navigate(R.id.action_loginPageFragment_to_homePageEmptyCase)
        }

        return root
    }
}